### PR TITLE
Add partial dependency plots

### PR DESCRIPTION
**Pull request by wamsiv**
Adding PD plots for `xgboost notebook`

### CODE REVIEW

File: notebooks/Customer_Automotive_Churn_Predictions.ipynb [hunk 1]
```diff
@@ -0,0 +1,1866 @@
...
+     "text": [
+      "Connection(mapd://mapd:***@http://localhost:9090/mapd?protocol=http)\n"
+     ]
+    }
+   ],
+   "source": [
+    "dbname    = 'mapd'\n",
+    "username  = 'mapd'\n",
+    "password  = 'HyperInteractive'\n",
+    "hostname  = 'localhost'\n",
```
Comment (andrewseidl) [inline notebooks/Customer_Automotive_Churn_Predictions.ipynb]: Might want to have a note saying to change this to `mapd-core` if running via `nvidia-docker-compose`

File: notebooks/Customer_Automotive_Churn_Predictions.ipynb [hunk 2]
```diff
@@ -0,0 +1,1866 @@
...
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Number of Columns: 23\n"
+     ]
+    }
+   ],
+   "source": [
+    "table= 'churn'\n",
```
Comment (andrewseidl) [inline notebooks/Customer_Automotive_Churn_Predictions.ipynb]: Is the `churn` data public? If so, should have a section showing how to import it into MapD.

File: notebooks/Customer_Automotive_Churn_Predictions.ipynb [hunk 3]
```diff
@@ -0,0 +1,1866 @@
...
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from xgboost import plot_importance\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
```
Comment (andrewseidl) [inline notebooks/Customer_Automotive_Churn_Predictions.ipynb]: `matplotlib` isn't listed in the conda env, so this import might fail.

Comment (wamsiv) [reply notebooks/Customer_Automotive_Churn_Predictions.ipynb]: added it to env

File: notebooks/Customer_Automotive_Churn_Predictions.ipynb (hunk 2 above)
Comment (wamsiv) [reply notebooks/Customer_Automotive_Churn_Predictions.ipynb]: added the URL

Comment (wamsiv) [reply notebooks/Customer_Automotive_Churn_Predictions.ipynb]: Added a comment about the privacy of dataset.

**Pull request merged**